src/main/java/io/jenkins/plugins/gitlabbranchsource/GitLabWebHookListener.java

```diff
@@ -19,6 +22,9 @@
     // GitLab API caching timeout used for branches endpoint
     private static final long GITLAB_CACHING_TIMEOUT = 30;
 
+    private List<String> ignoreActions = Arrays.asList("unapproved", "approved");
```

Comment:
Hi! Arrays in 25, 26 lines looks like need to be configured per project

Reply:
Thank you for the comments. I can look into that. 

Couple of questions:

Why would you ever want to run a pipeline just because the MR is approved, though? And especially if it's unapproved.

And you would always want it to run on open and reopen because gitlab resets the pipeline status.

By having those arrays and logic, it's just ensuring the current behavior is maintained while still allowing the title updates etc to be ignored.

Reply:
Sometimes we want to build our code by commets. Like in chatOps in Jenkins X. 
We can change code, but not in this repo.

Reply:
Ah I see. Comments are still working as normal: https://github.com/jenkinsci/gitlab-branch-source-plugin/blob/31f57008997f8d9ef3ac0793d0dd5b9015ce6666/src/main/java/io/jenkins/plugins/gitlabbranchsource/GitLabWebHookListener.java#L35

I will see if I can make this configurable in the GitLab project options though.

Reply:
Completely refactored it to make it configurable by the user.